eggplant/membership/models/invitation.py

```diff
@@ -47,20 +48,33 @@ def __str__(self):
     dispatch_uid='membership-invitation-send_email_invitation'
 )
 def send_email_invitation(sender, instance, created, **kwargs):
+
+    department = instance.department.name
+
     if created:
-        subject = 'You have been invited to Eggplant!'
-        to_addrs = [instance.email]
+        subject = _('You have been invited to join {department}!').format(
+            department=department
+        )
+        to_addrs = [instance.email, ]
         invite_url = absolute_url_reverse(
             url_name='eggplant:membership:accept_invitation',
             kwargs=dict(
                 verification_key=instance.verification_key.hex
             )
         )
-        body = """
-            Please click the following link to confirm:
-            {invite_url}
-
-        """.format(invite_url=invite_url)
+        body = _(
```

Comment:
Agreed, I'm opening an issue about getting a base email class that we can extend from... it will lead to bad places if we keep this pattern of string formatting for text email, manually typing in email signatures etc... bwadr